packages/vscode-dependencies-validation/package.json

```diff
@@ -20,13 +20,18 @@
     "ci": "npm-run-all clean compile bundle",
     "clean": "rimraf ./dist *.vsix",
     "compile": "tsc -p ./",
+    "watch": "tsc -p ./ --watch",
     "test": "mocha",
     "coverage": "nyc mocha",
     "bundle": "webpack --mode production",
     "package": "node ./scripts/package-vsix.js"
   },
   "dependencies": {
-    "@sap-devx/npm-dependencies-validation": "file:../npm-dependencies-validation"
+    "@sap-devx/npm-dependencies-validation": "file:../npm-dependencies-validation",
+    "vfile": "4.2.1",
+    "vfile-location": "3.2.0",
```

Comment:
afaik we are not using the vfile/vfile-location/jsonc-parser (yet)

Reply:
removed